composer.json

```diff
@@ -16,6 +16,7 @@
         "league/csv": "^9.0",
         "nelmio/cors-bundle": "^2.0",
         "nesbot/carbon": "^2.11",
+        "oat-sa/bundle-lti1p3": "^3.0",
```

Comment:
please req`"oat-sa/bundle-lti1p3": "^3.1",`, you'll need the scope compliant firewalls, released in `3.1.0`

see https://github.com/oat-sa/bundle-lti1p3/releases/tag/3.1.0 for details